bylaws.html

```diff
@@ -117,31 +117,31 @@ <h3 id="pmc">2.4. Project Management Committee</h3>
 
 <p>While security issues and release management are the responsibility of the PMC, the PMC typically delegates this to committers.
 
 <h3 id="chair">2.5. Chair</h3>
 
-<p><strong>The project Chair is a PMC member responsible for Foundation level administrative tasks.</strong> It is not a technical leadership position, meaning the Chair has no special say in ordinary project decisions. But we do think of it as a cultural leadership position. Accordingly, position on cultural issues is something to consider when electing a Chair.
+<p><strong>The project chair is a PMC member responsible for foundation-level administrative tasks.</strong> It is not a technical leadership position, meaning the chair has no special say in ordinary project decisions. However, we do think of it as a cultural leadership position. Accordingly, position on cultural issues is something to consider when electing a chair.
 
-<p>The Chair is elected by the PMC but appointed by the ASF Board via a Board resolution. The Chair is an officer of the Apache Software Foundation (with an official title of <em>Vice President, Apache CouchDB</em>) and has primary responsibility to the Board for the management of the project. The Chair is the eyes and ears of the Board, and reports quarterly on developments within the project.
+<p>The PMC elect the chair, but the board must officially appoint them with a board resolution. The chair is an officer of the ASF (with an official title of <em>Vice President, Apache CouchDB</em>) and has a primary responsibility to the board for the management of the project. The chair is the eyes and ears of the board, and reports quarterly on developments within the project.
 
-<p>Remember that, as in any meeting, the Chair is a facilitator and their role within the PMC is to ensure that everyone has a chance to be heard and to enable meetings to flow smoothly.
+<p>The chair is a facilitator, and their role within the PMC is to ensure that everyone has a chance to be heard and to enable meetings to flow smoothly.
 
-<p>If the current Chair resigns, or the term of the Chair expires, the PMC will hold a Chair election.
+<p>If the current chair resigns, or the term of the chair expires, the PMC will hold a chair election.
 
-<p>The Chair has a 12 month term. The intention of this term is to allow for a rotation of the role amongst the PMC members. This does not prohibit the PMC from selecting the same Chair to serve consecutive terms.
+<p>The chair has a 12-month term. The allows for a rotation of the role amongst the PMC members. However, the PMC may select the same chair to serve consecutive terms.
```

Comment:
should be "This allows for"?

Reply:
Also, I think we're behind on holding this election. Oops.